contracts/PeriodicTokenVesting.sol

```diff
@@ -35,6 +35,9 @@ contract PeriodicTokenVesting is OwnableUpgradeable, PausableUpgradeable {
     /// @dev The duration in seconds of a vesting period.
     uint256 private periodDuration;
```

Comment:
I would name it `period`

Reply:
Adding duration seems fitting as the name itself suggests it involves time. 

Calling it only period might be misleading as "period" could be anything, a counter, and index, etc.